website/docs/r/codeartifact_repository.html.markdown

```diff
@@ -88,8 +88,9 @@ The following arguments are supported:
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - The Name of the repository.
+* `id` - The ARN of the repository.
```

Comment:
This looks good based on the [underlying code](https://github.com/hashicorp/terraform-provider-aws/blob/bf0941493da9a86098ab735fbe2150189db829ad/aws/resource_aws_codeartifact_repository.go#L127) 👍